tests_tf/test_attacks.py

```diff
@@ -6,16 +6,29 @@
 import unittest
 import numpy as np
 
+from cleverhans.attacks import FastGradientMethod
+from cleverhans.attacks import BasicIterativeMethod
 from cleverhans.attacks import VirtualAdversarialMethod
+from cleverhans.attacks import SaliencyMapMethod
 
+import time
 
-class TestVirtualAdversarialMethod(unittest.TestCase):
+class CleverHansTest(unittest.TestCase):
     def setUp(self):
+        self.test_start = time.time()
+        
+    def tearDown(self):
+        print(self.id(), "took", time.time() - self.test_start, "seconds")
+
+
+class TestVirtualAdversarialMethod(CleverHansTest):
+    def setUp(self):
+        super(TestVirtualAdversarialMethod, self).setUp()
         import tensorflow as tf
         import tensorflow.contrib.slim as slim
 
         def dummy_model(x):
-            net = slim.fully_connected(x, 600)
+            net = slim.fully_connected(x, 60)
```

Comment:
Just a question (not a change request): was this changed because the test took too long?